README.md

```diff
@@ -46,6 +46,23 @@ Unit tests can be running using the following `npm` command:
 ```bash
 npm run test
 ```
+
+## Usage
+
+FQM-Testify is a React-based web application for analyzing FHIR Electronic Clinical Quality Measures (eCQMs) through the creation of test FHIR Patients and resources.
+
+### Creating a Test Case
+
+For instructions on starting the app, check [here](README.md#L20). After starting up the app, begin by uploading a FHIR Measure Bundle from your local machine. The measurement period start and end pickers will update to reflect the effectivePeriod of the Measure resource in the uploaded bundle.
+
+Test patients can be added to the app by clicking on either the "Create Test Patient" button, or the "Import Test Patient(s)" button. The former will open a code editor modal with a pre-populated, randomly-generated FHIR Patient. The latter will open a file dropzone which accepts JSON files of FHIR Patient Bundles and .zip files composed of FHIR Patient Bundles.
```

Comment:
I think this might be more readable if we break this paragraph up into two sections, one for creating a new test patient and one for importing test patient(s). Maybe we can say something like "There are two ways that a user can add patients to the test case:" and then have two level 4 headings that say "Create Test Patient" and "Import Test Patient(s)". This might also make it easier to explain the different ways in which test patients can be imported.